packages/core/src/endowmentsToolkit.js

```
@@ -751,6 +751,12 @@ function defaultCreateFunctionWrapper(sourceValue, unwrapTest, unwrapTo) {
     newValue,
     Object.getOwnPropertyDescriptors(sourceValue)
   )
+  // Global functions can have static fields on a prototype. eg. Uint8Array.from
+  if (
+    Reflect.getPrototypeOf(newValue) !== Reflect.getPrototypeOf(sourceValue)
+  ) {
+    Object.setPrototypeOf(newValue, Reflect.getPrototypeOf(sourceValue))
```

Is `Object.setPrototypeOf` necessary instead of `Reflect.setPrototypeOf`?  Does it matter?  If it does, please explain.

---

Object.setPrototypeOf throws on failure. Reflect.setPrototypeOf fails silently. I'd keep it as is.
```
> const a = {}
undefined
> Object.freeze(a)
{}
> Reflect.setPrototypeOf(a, 1)
Uncaught TypeError: Object prototype may only be an Object or null: 1
    at Reflect.setPrototypeOf (<anonymous>)
> Reflect.setPrototypeOf(a, {})
false
> Object.setPrototypeOf(a, {})
Uncaught TypeError: #<Object> is not extensible
    at Function.setPrototypeOf (<anonymous>)
```